purchase.adoc

```diff
@@ -443,6 +429,96 @@ purchaseUUID:: The UUID of the purchase
 }
 ```
 
+### Payments
+
+TODO: Add some more overview info
+
+._example: Version 2 purchase with one card, one cash and one invoice payment_
+----
+{
+  ...
+  "payments": [
+    {
+      "type": "IZETTLE_CARD",
+      ...
+    },
+    {
+      "type": "IZETTLE_CASH",
+      ...
+    },
+    {
+      "type": "IZETTLE_INVOICE",
+      ...
+    },
+  ],
+  ...
+}
+----
+
+In addition to `IZETTLE_CARD`, `IZETTLE_CARD_ONLINE`, `IZETTLE_CASH` and `IZETTLE_INVOICE`, there is also payment types of `SWISH`, `VIPPS`, `MOBILE_PAY`, `PAYPAL`, `STORE_CREDIT` and `CUSTOM`.
```

Comment:
How about GIFTCARD, KLARNA etc?  How much of the future payment should we expose here?

Reply:
Good point! My spontaneous thought is that we should probably be cautious about sharing info on not yet released products, so I would suggest adding the payment types when more officially launched.